.pvs-suppress

```diff
@@ -360,6 +360,30 @@
             "ErrorCode": "V801",
             "FileName": "mouseif_ps2_bios.cpp",
             "Message": "Decreased performance. It is better to redefine the second function argument as a reference. Consider replacing 'const .. new_buttons_all' with 'const .. &new_buttons_all'."
+        },
+        {
+            "CodeCurrent": 481966618,
+            "CodeNext": 368006577,
+            "CodePrev": 0,
+            "ErrorCode": "V813",
+            "FileName": "dos_files.cpp",
+            "Message": "Decreased performance. The 'attributes' argument should probably be rendered as a constant reference."
+        },
+        {
+            "CodeCurrent": 4048577584,
+            "CodeNext": 4015905278,
+            "CodePrev": 3396962584,
+            "ErrorCode": "V813",
+            "FileName": "dos_files.cpp",
+            "Message": "Decreased performance. The 'createAttr' argument should probably be rendered as a constant reference."
+        },
+        {
+            "CodeCurrent": 205934690,
+            "CodeNext": 123,
+            "CodePrev": 0,
+            "ErrorCode": "V813",
+            "FileName": "drive_overlay.cpp",
+            "Message": "Decreased performance. The 'attr' argument should probably be rendered as a constant reference."
         }
```

Comment:
I can't remember if we wanted to knock these out entirely?

@FeralChild64 , feel free to alternative add V813 to `.github/workflows/pvs-studio.yml`:


```yaml
      - name: Analyze
        run: |
          set -xeu
          log="pvs-report.log"
          general_criteria="GA:1,2;64:1;OP:1,2,3;CS:1"
          stamp="$(date +'%Y-%m-%d_T%H%M')-${GITHUB_SHA:0:8}"
          reportdir="pvs-report/pvs-report-${stamp}"
          disable_warnings="V002,V801,V802,V826,V1042,V1071,V2008"
```

Reply:
OK, added V813 to the list, also removed obsolete V801 suppressions.